examples/test/MeasureValueFilterComponent_test.js

```diff
@@ -71,3 +74,19 @@ test("should filter dropdown with custom button", async t => {
     await checkCellValue(t, pivotTableSelector, "Montgomery", CELL_0_0);
     await checkCellValue(t, pivotTableSelector, "16,077,036", CELL_0_1);
 });
+
+test("should filter the data with treat null value as zero", async t => {
+    const pivotTableSelector = Selector(".s-measure-value-filter-example-5");
+    await t.click(comparisonFilterDropdownSelector.nth(4));
+    await t.click(comparisonFilterOperatorDropdownSelector);
+    await t.click(lessThanIconSelector);
+    await t.typeText(comparisonInputSelector, "10");
+    await t.click(applySelector);
+
+    await waitForPivotTableStopLoading(t, pivotTableSelector);
+    // check applied filter
```

Comment:
Either remove this comment or remove the blank line after it (or put it above it) as the comment belongs to the next block.